lib/fuel/configuration.rb

```diff
@@ -3,7 +3,7 @@ class Configuration
 
     # What kind of comments do you want to add to your blog ? (:active_record, :disqus or :no)
     attr_accessor :layout, :blog_title, :disqus_name, :twitter, :username, :password, :helpers, :paginates_per,
-        :featured_image_settings, :avatar_settings, :facebook_app_id, :site_name, :blog_description, :tags, :logo
+        :featured_image_settings, :avatar_settings, :facebook_app_id, :site_name, :blog_description, :tags, :logo, :aws_bucket, :aws_access_key, :aws_secret_access_key, :paperclip_defaults
```

Comment:
Line is too long. [188/80]<br>Align the parameters of a method call if they span more than one line.